### PR TITLE
modified index.html and index.css files

### PR DESCRIPTION
**Pull request by chilara**
done

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -1,11 +1,338 @@
...
+h5{
+    color: #ffffff;
+    
+}
+
+h4{
+    color: #ffffff;
+    
+}
+
```
Comment (andrewinsoul) [inline index.css:117]: h1, h5 and h4 have similar styles. A more concise way of styling them is:
h1, h5, h4 {
   color: #ffffff
}

**Review by andrewinsoul — CHANGES_REQUESTED** [review]

**Comment by chilara** [comment]
Roger that boss.

**Review by andrewinsoul — APPROVED** [review]
u can now merge

**Review by andrewinsoul — APPROVED** [review]

**Comment by chilara** [comment]
Awesome, thanks!

**Pull request merged**